common/src/main/java/com/wynntils/core/features/UserFeature.java

```diff
@@ -33,13 +33,20 @@ public final void updateConfigOption(ConfigHolder configHolder) {
     /** Updates the feature's enabled/disabled state to match the user's setting, if necessary */
     public final void tryUserToggle() {
         if (userEnabled) {
-            tryEnable();
+            enable();
         } else {
-            tryDisable();
+            disable();
         }
     }
 
     public void setUserEnabled(boolean newState) {
-        this.userEnabled = newState;
+        Optional<ConfigHolder> opt = getConfigOptionFromString("userEnabled");
```

Comment:
What is the gain of looking up the config instead of using the field directly? I'm really confused here.

Reply:
It actually saves, thought this had unintended side affects, so this will be reverted soon

Reply:
If you only set the field, you won't call the onConfigUpdate with the userEnabled config